agent/apps/ai_guardrails/app_code/permissions.py

```
@@ -0,0 +1,29 @@
+
+class IsGuardrailsAdminOrReadOnly(BasePermission):
+    """
+    GET/HEAD/OPTIONS آزاد، بقیه فقط برای ادمین.
+    """
+    def has_permission(self, request, view) -> bool:
+        if request.method in SAFE_METHODS:
+            return True
+        user = getattr(request, "user", None)
+        return bool(user and getattr(user, "is_staff", False))
+
+__all__ = ["IsGuardrailsAuthenticated", "IsGuardrailsAdminOrReadOnly"]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)